src/components/Layout/Header/ActionCenter/components/Notifications/LimitOrderNotification.tsx

```diff
@@ -0,0 +1,171 @@
+
+type SwapNotificationProps = {
+  handleClick: () => void
+  action: LimitOrderAction
+} & RenderProps
+
+const toastHoverProps = {
+  transform: 'translateY(-2px)',
+}
+
+const divider = <RawText color='text.subtle'>•</RawText>
+
+export const LimitOrderNotification = ({ handleClick, action, onClose }: SwapNotificationProps) => {
+  const crossBgColor = useColorModeValue('gray.850', 'white')
+  const crossColor = useColorModeValue('white', 'gray.850')
+
+  const buttonHoverProps = useMemo(
+    () => ({
+      opacity: 0.8,
+      bg: crossBgColor,
+      color: crossColor,
+    }),
+    [crossBgColor, crossColor],
+  )
+
+  const limitOrderNotificationTranslationComponents: TextPropTypes['components'] = useMemo(() => {
+    if (action.status === ActionStatus.Complete) {
+      return {
+        sellAmountAndSymbol: (
+          <Amount.Crypto
+            value={action.limitOrderMetadata.executedSellAmountCryptoPrecision}
+            symbol={action.limitOrderMetadata.sellAsset.symbol}
+            fontSize='sm'
+            fontWeight='bold'
+            maximumFractionDigits={6}
+            omitDecimalTrailingZeros
+            display='inline'
+          />
+        ),
+        buyAmountAndSymbol: (
+          <Amount.Crypto
+            value={action.limitOrderMetadata.executedBuyAmountCryptoPrecision}
+            symbol={action.limitOrderMetadata.buyAsset.symbol}
+            fontSize='sm'
+            fontWeight='bold'
+            maximumFractionDigits={6}
+            omitDecimalTrailingZeros
+            display='inline'
+          />
+        ),
+      }
+    }
+
+    return {
+      sellAmountAndSymbol: (
+        <Amount.Crypto
+          value={action.limitOrderMetadata.sellAmountCryptoPrecision}
+          symbol={action.limitOrderMetadata.sellAsset.symbol}
+          fontSize='sm'
+          fontWeight='bold'
+          maximumFractionDigits={6}
+          omitDecimalTrailingZeros
+          display='inline'
+        />
+      ),
+      buyAmountAndSymbol: (
+        <Amount.Crypto
+          value={action.limitOrderMetadata.buyAmountCryptoPrecision}
+          symbol={action.limitOrderMetadata.buyAsset.symbol}
+          fontSize='sm'
+          fontWeight='bold'
+          maximumFractionDigits={6}
+          omitDecimalTrailingZeros
+          display='inline'
+        />
+      ),
+    }
+  }, [action])
+
+  const limitOrderTitleTranslation = useMemo(() => {
+    if (!action) return 'notificationCenter.limitOrder.processing'
+    if (action.status === ActionStatus.Open) return 'notificationCenter.limitOrder.placed'
+    if (action.status === ActionStatus.Complete) return 'notificationCenter.limitOrder.complete'
+    if (action.status === ActionStatus.Expired) return 'notificationCenter.limitOrder.expired'
+    if (action.status === ActionStatus.Cancelled) return 'notificationCenter.limitOrder.cancelled'
+
+    return 'notificationCenter.limitOrder.placed'
+  }, [action])
+
+  const formattedDate = useMemo(() => {
```

Comment:
We probably shouldn't have this in the toast notification, only in the akschual one in drawer, feels odd, and `a minute ago` is wrong, dis now